.travis.yml

```diff
@@ -32,11 +33,9 @@ env:
   - LD_LIBRARY_PATH=/usr/local/lib
 
 addons:
-  apt:
+  apt: &common_apt
     config:
       retries: true
-    sources: &common_sources
-    - ubuntu-toolchain-r-test
```

Comment:
If we want to update to GCC 9, we need to add this again. So I'd maybe rather update to GCC 9 right now instead of removing this and adding it again later?